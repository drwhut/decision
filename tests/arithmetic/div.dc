Start~#1

< Integer dividing 2 positive integers >
Div(725, 42)~#2
Print(#1, #2)~#3

< Integer dividing a negative integer from a positive integer >
Div(15, -5)~#4
Print(#3, #4)~#5

< Integer dividing a positive integer from a negative integer >
Div(-20, 100)~#6
Print(#5, #6)~#7

< Integer dividing 2 negative integers >
Div(-5, -7)~#8
Print(#7, #8)~#9

< Integer dividing 2 positive floats >
Div(0.5762, 0.1399)~#10
Print(#9, #10)~#11

< Integer dividing a negative float from a positive float >
Div(123.456, -64.32)~#12
Print(#11, #12)~#13

< Integer dividing a positive float from a negative float >
Div(-7.09, 10.12)~#14
Print(#13, #14)~#15

< Integer dividing 2 negative floats >
Div(-12000.0, -6014.25)~#16
Print(#15, #16)~#17

< Integer dividing a float from an integer >
Div(12, 5.3)~#18
Print(#17, #18)~#19

< Integer dividing an integer from a float >
Div(4.23, 1)~#20
Print(#19, #20)~#21