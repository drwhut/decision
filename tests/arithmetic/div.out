17
-3
0
0
4
-1
0
1
2
4
